lib/interest_days/calculation/icma_act_act.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module InterestDays
+  module Calculation
+    # ISDA Act Act Convention calculation
+    class IcmaActAct < Base
+      DAYS_IN_YEAR = 365
+      DAYS_IN_LEAP_YEAR = 366
+
+      def day_count_factor
+        return days.fdiv(start_date_days_in_year) if start_date_and_end_date_in_same_year?
+
+        if years_in_between_factor > 1
+          return start_date_period_factor + years_in_between_factor + end_date_period_factor
+        end
```

Comment:
Really nice specs! I would add an integration test for years in between :)

Reply:
done